test/Kestrel.Interop.FunctionalTests/SkipIfChromeUnavailableAttribute.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Microsoft.AspNetCore.Testing.xunit;
+
+namespace Interop.FunctionalTests
+{
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Class, AllowMultiple = false)]
+    public class SkipIfChromeUnavailableAttribute : Attribute, ITestCondition
+    {
+        public bool IsMet => !string.IsNullOrEmpty(Environment.GetEnvironmentVariable("CI")) || File.Exists(ChromeConstants.ExecutablePath);
```

Comment:
We want to force the tests to run for CI environments. However, in dev scenarios, the tests should only run if Chrome is installed.